test/integration/ink_editor_spec.js

```diff
@@ -20,11 +20,11 @@ const {
   waitForStorageEntries,
 } = require("./test_utils.js");
 
-const waitForClick = async page =>
+const waitForClick = page =>
   page.evaluate(
     () =>
       new Promise(resolve => {
-        window.addEventListener("click", resolve, { once: true });
+        window.addEventListener("pointerup", resolve, { once: true });
```

Comment:
Please add a (short) comment explaining why we use "pointerup" here, since otherwise it's a bit confusing why a method named `waitForClick` waits for a "pointerup" event.